### PR TITLE
Fix: More than one NULL to separate commits. #1460

### PR DESCRIPTION
**Pull request by ylecuyer**
We use the `-z` flag within git log to separate commits with NULL chars instead of new lines

In this case for example there is no one but two NULL chars before the next commits thus failing with the previous code

### CODE REVIEW

File: source/git-parser.js [hunk 1]
```diff
@@ -162,7 +162,7 @@ exports.parseGitLog = (data) => {
       parser = parseFileChanges;
       return;
     }
-    if (rows[index + 1] && rows[index + 1].indexOf('\x00commit ') == 0) {
+    if (rows[index + 1] && rows[index + 1].match(/^\u0000+commit/) != null) {
```
Comment (campersau) [inline source/git-parser.js:165]: Would `test` instead of `match` also work here?

Comment (ylecuyer) [reply source/git-parser.js:165]: Indeed it will be better, I'll update the PR

**Review by campersau — APPROVED** [review]
Looks good thanks!

Just curious, what is #1560 referring to?

**Comment by ylecuyer** [comment]
> Looks good thanks!
> 
> Just curious, what is #1560 referring to?

It is a mistake, it should read #1460 I have udpated the title :)

**Pull request merged**